freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/UserSyncService.java

```diff
@@ -355,13 +368,54 @@ private SyncStatusDetail internalSynchronizeStack(Stack stack, UmsUsersState ums
         }
     }
 
+    private SyncStatusDetail internalSynchronizeStackForDeleteUser(Stack stack, String deletedWorkloadUser) {
+        MDCBuilder.buildMdcContext(stack);
+        String environmentCrn = stack.getEnvironmentCrn();
+        Multimap<String, String> warnings = ArrayListMultimap.create();
+        try {
+            FreeIpaClient freeIpaClient = freeIpaClientFactory.getFreeIpaClientForStack(stack);
+
+            LOGGER.debug("Starting {} for environment {} and deleted user {} ...", LogEvent.USER_SYNC_DELETE, environmentCrn, deletedWorkloadUser);
+
+            LogEvent logEvent = LogEvent.RETRIEVE_PARTIAL_IPA_STATE;
```

Comment:
I think you should either delete `logEvent` variable or use it in the next line also

Reply:
Deleted